projects/core/src/occ/adapters/asm/occ-asm.adapter.spec.ts

```diff
@@ -43,39 +45,48 @@ class MockBaseSiteService {
   }
 }
 
-describe('CustomerService', () => {
-  let customerService: CustomerService;
+export class MockOccEndpointsService {
+  getRawEndpoint(endpoint: string): string {
+    return endpoint;
+  }
+}
+
+describe('OccAsmAdapter', () => {
+  let occAsmAdapter: OccAsmAdapter;
+  let converterService: ConverterService;
   let httpMock: HttpTestingController;
+  let occEnpointsService: OccEndpointsService;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [HttpClientTestingModule],
       providers: [
-        CustomerService,
+        OccAsmAdapter,
         { provide: BaseSiteService, useClass: MockBaseSiteService },
         { provide: AsmConfig, useValue: MockAsmConfig },
+        { provide: OccEndpointsService, useClass: MockOccEndpointsService },
       ],
     });
 
-    customerService = TestBed.get(CustomerService);
-    httpMock = TestBed.get(HttpTestingController as Type<
-      HttpTestingController
-    >);
+    occAsmAdapter = TestBed.get(OccAsmAdapter);
```

Comment:
This might be silly but I just thought about it. You know a few weeks back there was this big push to put `TestBest.get(xxx as Type<XXX>);` Do we still do that?

Reply:
I'm not aware of this.  There does not seem to be lint warnings produced by this syntax. ( see CI logs )